examples/metrics/src/main/java/io/opentelemetry/example/metrics/ObserverMeterExample.java

```diff
@@ -0,0 +1,25 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.metrics.LongValueObserver;
+import io.opentelemetry.metrics.LongValueObserver.ResultLongValueObserver;
+import io.opentelemetry.metrics.Meter;
+
+public class ObserverMeterExample {
+
+  public static void main(String[] args) {
+    Meter sampleMeter = OpenTelemetry.getMeterProvider().get("sample", "0.1");
```

Comment:
I think it would be helpful to have the meter name reflect reommended meter naming standards, which would mean this should probably be something like "io.opentelemetry.example.metrics" with a version of 0.5, maybe